NewBank/newbank/client/ExampleClient.java

```diff
@@ -53,6 +53,21 @@ public void run() {
     }
 
     public static void main(String[] args) throws UnknownHostException, IOException, InterruptedException {
-        new ExampleClient("localhost", 14002).start();
+        Boolean retryConnecting = true;
+        while(retryConnecting){
+            try {
+                new ExampleClient("localhost", 14002).start();
+                retryConnecting = false; // already connected
+            } catch (IOException e){
+                System.out.println("Failed to connect to the bank, do you want to retry? [y/n]");
+                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+                String response = reader.readLine();
+                if(response.equals("y\n")){
+                    retryConnecting = false;
+                }
```

Comment:
All looks good, I just have one query about the retry connection:
When I tried inputting n, it responded by retrying the connection (same as inputting y) instead of closing the program.
Just wondering what should occur in this case in everyone else's opinion?